filters/include/pcl/filters/uniform_sampling.h

```diff
@@ -72,7 +72,8 @@ namespace pcl
       typedef boost::shared_ptr<const UniformSampling<PointT> > ConstPtr;
 
       /** \brief Empty constructor. */
-      UniformSampling () :
+      UniformSampling (bool extract_removed_indices = false) :
```

Comment:
Maybe we just have two constructors? One parameterless, one with compulsory bool.